op-chain-ops/crossdomain/precheck.go

```diff
@@ -26,14 +32,15 @@ func PreCheckWithdrawals(db *state.StateDB, withdrawals []*LegacyWithdrawal) ([]
 
 	// Build a mapping of the slots of all messages actually sent in the legacy system.
 	var count int
+	var innerErr error
 	slotsAct := make(map[common.Hash]bool)
 	err := db.ForEachStorage(predeploys.LegacyMessagePasserAddr, func(key, value common.Hash) bool {
 		// When a message is inserted into the LegacyMessagePasser, it is stored with the value
 		// of the ABI encoding of "true". Although there should not be any other storage slots, we
 		// can safely ignore anything that is not "true".
 		if value != abiTrue {
 			// Should not happen!
-			log.Error("found unknown slot in LegacyMessagePasser", "key", key.String(), "val", value.String())
+			innerErr = ErrUnknownSlotInMessagePasser
```

Comment:
Any reason to just not return `false` and not use `innerErr`?

Reply:
Unless false just stops the iteration and doesn't return an error?

Reply:
Correct, `false` just stops iteration and doesn't return an error.